cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreTableManager.java

```diff
@@ -99,6 +99,7 @@ public ExploreTableManager(ExploreService exploreService,
                              ExploreTableNaming tableNaming,
                              Configuration hConf) {
     this.exploreService = exploreService;
+    // TODO: need impersonation in SystemDatasetInstantiator#getDataset?
```

Comment:
It doesn't do impersonation. Usually, its called from program containers, where impersonation is not required.